src/stdlib/mod.rs

```diff
@@ -134,6 +135,7 @@ fn assert_eq<B: Backend>(
 /// Asserts that a condition is true.
 fn assert<B: Backend>(
     compiler: &mut CircuitWriter<B>,
+    _generics: &GenericParameters,
```

Comment:
I'm wondering why we're doing this

Reply:
these builtins don't need generics.
but the others would need them, such as bit functions.